File: lol/src/main/kotlin/com/your/lol/dto/riot/ParticipantDto.java
package com.your.lol.dto.riot;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class ParticipantDto {
    private String summonerName;
    private int teamId;
    private String championName;
    private long gameDuration;
    private ChallengesDto challenges;

    public boolean equalsId(String summonerName) {
        return this.summonerName.equals(summonerName);
    }
}

File: lol/src/main/kotlin/com/your/lol/dto/riot/ParticipantDto.kt
package com.your.lol.dto.riot

class ParticipantDto(
        val summonerName: String,
        val teamId: Int,
        val championName: String,
        val gameDuration: Long,
        val challenges: ChallengesDto? = null) {

    private constructor() : this("", 0, "", 0, null)

    fun equalsId(summonerName: String): Boolean {
        return this.summonerName == summonerName
    }
}